crates/swc_css_parser/src/parser/selector/mod.rs

```diff
@@ -408,6 +443,40 @@ where
 
         let mut nesting_selector = None;
 
+        if self.config.css_modules && is!(self, ":") && peeked_is_one_of!(self, "local", "global") {
+            bump!(self);
+
+            let kind: Ident = self.parse()?;
+
+            if eat!(self, "(") {
+                let selectors = self.parse_as::<SelectorList>()?;
```

Comment:
Inside `local` and `global` is not `SelectorList`, it is not valid to use `a, b` inside `:local`, because every local contain only compound selector